core/server/models/webhook.js

```diff
@@ -0,0 +1,66 @@
+var ghostBookshelf = require('./base'),
+    events = require('../events'),
+    Promise = require('bluebird'),
+    Webhook,
+    Webhooks;
+
+Webhook = ghostBookshelf.Model.extend({
+    tableName: 'webhooks',
+
+    emitChange: function emitChange(event) {
+        events.emit('webhook' + '.' + event, this);
+    },
+
+    onCreated: function onCreated(model) {
+        model.emitChange('added');
+    },
+
+    onUpdated: function onUpdated(model) {
+        model.emitChange('edited');
+    },
+
+    onDestroyed: function onDestroyed(model) {
+        model.emitChange('deleted');
+    }
+}, {
+    findAllByEvent: function findAllByEvent(event, options) {
+        var webhooksCollection = Webhooks.forge();
+
+        options = this.filterOptions(options, 'findAll');
+
+        return webhooksCollection
+            .query({where: {event: event}})
```

Comment:
Is the object query syntax ok? I see in _most_ other places we use the `.query('where', 'event', '=', event)` pattern instead, is that preferred?

Reply:
The pattern with the explicit operator is slightly more readable, IMO.

I believe you can also do the where directly, e.g. `webhooksCollection.where({event: event})` or `.where('event', '=', event)` ... there's a million ways to "phrase" bookshelf and knex together which is not a good thing.

More importantly here, completely contextual knowledge, but bookshelf Collections are near-useless and probably going away in the near future, if not from bookshelf itself, very likely from Ghost.

`this.forge().where(event, '=', event).fetchAll(options)` should do the exact same thing.

I now realise the destroyBy examples in the token model are all different 🙈